package.json

```diff
@@ -47,5 +47,8 @@
     "jest": "^25.1.0",
     "prettier": "^1.19.1",
     "rollup": "^1.31.1"
+  },
+  "dependencies": {
+    "jest-matcher-deep-close-to": "^1.3.0"
```

Comment:
Please put this as a devDependenciy:
`npm uninstall jest-matcher-deep-close-to`
`npm i jest-matcher-deep-close-to --save-dev`